erpnext/stock/doctype/material_request_item/material_request_item.json

```diff
@@ -474,23 +474,23 @@
    "options": "Batch",
    "read_only": 1
   },
-  {
-   "fieldname": "sticker_details",
-   "fieldtype": "Long Text",
-   "label": "Sticker Details",
-   "read_only": 1
-  },
   {
    "collapsible": 1,
-   "collapsible_depends_on": "eval:doc.request_for!='Stickers'",
+   "collapsible_depends_on": "eval:doc.request_for!='Labes'",
    "fieldname": "sb_batch",
    "fieldtype": "Section Break",
    "label": "Batch Details"
+  },
+  {
+   "fieldname": "label_details",
```

Comment:
You'll have to include this in the patch, move the data from the old field to this